setup.py

```diff
@@ -49,7 +49,10 @@
         "gunicorn",
         "six>=1.14.0",
         "bokeh",
-        "selenium"
+        "selenium",
+	"nltk",
```

Comment:
Please fix the indentation here. Also, why have `h5py` and `scipy` now been added?